src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private Random random = new Random();
+
+    public Colors randomColor() {
+        int color = random.nextInt(Colors.values().length);
```

Comment:
```suggestion
        int index = random.nextInt(Colors.values().length);
```

не информативное название переменной